jax/lax_linalg.py

```diff
@@ -158,14 +157,14 @@ def _cholesky_cpu_gpu_translation_rule(potrf_impl, c, operand):
 
 # Asymmetric eigendecomposition
 
-def eig_impl(operand):
-  return xla.apply_primitive(eig_p, operand)
+def eig_impl(operand, jobvl, jobvr):
```

Comment:
You should probably make these new arguments keyword-only, i.e., write this:
```
def eig_impl(operand, *, jobvl, jobvr):
```
Same for all the rules (e.g., the translation and abstract eval rules).

Reply:
On George's advice, I'm only dealing with `lapack.pyx` in this PR. The other points will be addressed in a further PR.